munkitools/munkitools2-autobuild.munki.recipe

```diff
@@ -18,6 +18,12 @@ https://munkibuilds.org/2.2.2.2416/munkitools-2.2.2.2416.pkg</string>
         <string>munkitools2</string>
         <key>MUNKI_CATALOG</key>
         <string>development</string>
+        <key>MUNKI_CATEGORY</key>
+        <string>munkitools</string>
+        <key>MUNKI_DEVELOPER</key>
+        <string>Greg Neagle</string>
```

Comment:
Sir Gregory Neagle, ruler of Munki land?

> On Jul 7, 2016, at 4:58 PM, Greg Neagle <notifications@github.com> wrote:
> 
> In munkitools/munkitools2-autobuild.munki.recipe:
> 
> > @@ -18,6 +18,12 @@ https://munkibuilds.org/2.2.2.2416/munkitools-2.2.2.2416.pkg</string>
> >          <string>munkitools2</string>
> >          <key>MUNKI_CATALOG</key>
> >          <string>development</string>
> > +        <key>MUNKI_CATEGORY</key>
> > +        <string>munkitools</string>
> > +        <key>MUNKI_DEVELOPER</key>
> > +        <string>Greg Neagle</string>
> Nope.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>